nodejs/BUILD

```diff
@@ -82,6 +82,7 @@ ts_project(
         ":node_modules/uuid",
         ":node_modules/@types/uuid",
     ],
+    transpiler = "tsc",
```

Comment:
```
Traceback (most recent call last):
        File "/private/var/tmp/_bazel_spare/afea01e34d46fe314768544eee28004b/external/aspect_rules_ts/ts/private/ts_project.bzl", line 246, column 21, in _ts_project_impl
                fail(transpiler_selection_required)
Error in fail: 
######## Required Transpiler Selection ########

You must select a transpiler for ts_project rules, which produces the .js outputs.

Please read https://docs.aspect.build/rules/aspect_rules_ts/docs/transpiler

##########################################################
```

Here I use the transpiler that @alexjpwalker used for `platform`, not sure if anything more special is needed. As I understand, it might affect the runtime speed of the `ts` application if the transpiler is incorrect for this specific app, and I don't think that we want to compare them for our rules. Maybe @alexjpwalker could help us verify it. 

> [tsc](https://docs.aspect.build/rules/aspect_rules_ts/docs/transpiler): This is the simplest configuration without additional dependencies. However, it's also the slowest.

+ I didn't want to put "standard" values in `.bazelrc`, I think it's much better to be explicit here.

Reply:
Approved by @alexjpwalker in https://github.com/vaticle/typedb-protocol/pull/202